create-node-api/src/main/scala/com/normation/plugins/createnodeapi/DataTypes.scala

```diff
@@ -142,8 +143,19 @@ object NodeTemplate {
 object Serialize {
   import net.liftweb.json._
 
-  implicit val formats = DefaultFormats
+  implicit val formats = DefaultFormats + new ConfigValueSerializer
 
+  class ConfigValueSerializer extends Serializer[ConfigValue] {
+    private val ConfigValueClass = classOf[ConfigValue]
+
+    def deserialize(implicit format: Formats): PartialFunction[(TypeInfo, JValue), ConfigValue] = {
+      case (TypeInfo(ConfigValueClass, _), json) => GenericProperty.fromJsonValue(json)
+    }
+
+    def serialize(implicit format: Formats): PartialFunction[Any, JValue] = {
+      case x: ConfigValue => JString(x.render())
```

Comment:
I'm not sure where it is used, but I fear it won't do what we want regarding escaping for js (ie: if `x` is json, then its jstring will be escaped json, so that parsing it back will be an actual string with escaped quotes etc).